### ISSUE TITLE
Error compiling video\videoGrabberExample / videoInput.lib / under Windows 11 / VS 2022  

### ISSUE THREAD

**Issue by moebiussurfing** [openframeworks/openFrameworks#7886]
Hello,
I am having these errors since recent updates from the main branch...

Any idea?

`K:\Documents\of_12\openFrameworks\examples\video\videoGrabberExample`

```.bash
Severity	Code	Description	Project	File	Line	Suppression State	Details
Error	MSB6006	"link.exe" exited with code 1319.	videoGrabberExample	C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.CppCommon.targets	1151		
Error	LNK1319	2 mismatches detected	videoGrabberExample	K:\Documents\of_12\openFrameworks\examples\video\videoGrabberExample\bin\videoGrabberExample_debug.exe	1		
Error	LNK2038	mismatch detected for 'RuntimeLibrary': value 'MD_DynamicRelease' doesn't match value 'MDd_DynamicDebug' in main.obj	videoGrabberExample	K:\Documents\of_12\openFrameworks\examples\video\videoGrabberExample\videoInput.lib(videoInput.obj)	1		
Error	LNK2038	mismatch detected for '_ITERATOR_DEBUG_LEVEL': value '0' doesn't match value '2' in main.obj	videoGrabberExample	K:\Documents\of_12\openFrameworks\examples\video\videoGrabberExample\videoInput.lib(videoInput.obj)	1		
```

![image](https://github.com/openframeworks/openFrameworks/assets/2106149/9e8af029-3a07-4d02-bb50-d59797a2dc43)

**Comment by danoli3** [comment]
Release mode will work

Debug mode fix:
- seems like this lib needs debug as using CRT stuff as well .
- recompile debug mode with crt flags for debug
- edit addon config include debug lib

**Comment by danoli3** [comment]
Fixed via #7888
